src/harness/testcases/WINNF_FT_S_SCS_testcase.py

```diff
@@ -64,3 +66,75 @@ def test_WINNF_FT_S_SCS_5(self):
     Checks that a CBSD registration with this configuration succeed.
     """
     self.doTestCipher('ECDHE-RSA-AES128-GCM-SHA256')
+
+  def generate_SCS_12_default_config(self, filename):
+    """Generates the WinnForum configuration for SCS.12"""
+    # Create the actual config for client cert/key path
+    
+    config = { 
+        'clientCert': self.getCertFilename("client_expired.cert"),
+        'clientKey':self.getCertFilename("client_expired.key")
+    }
+    writeConfig(filename, config)
+  
+  @configurable_testcase(generate_SCS_12_default_config)
+  def test_WINNF_FT_S_SCS_12(self,config_filename):
+    """Expired certificate presented during registration.
+
+    Checks that SAS UUT response with fatal alert message.
+    """
+    config = loadConfig(config_filename)
+    self.assertTlsHandshakeFailure(client_cert=config['client_cert_path'],
+                                   client_key=config['client_key_path'])
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_SCS_13(self):
+    """ Disallowed TLS method attempted during registration.
+
+    Checks that SAS UUT response with fatal alert message.
+    """
+    self.assertTlsHandshakeFailure(ssl_method=SSL.TLSv1_1_METHOD)
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_SCS_14(self):
+    """Invalid ciphersuite presented during registration.
+
+    Checks that SAS UUT response with fatal alert message.
+    """
+    self.assertTlsHandshakeFailure(ciphers='ECDHE-RSA-AES256-GCM-SHA384')
+
+  def generate_SCS_15_default_config(self, filename):
+    """ Generates the WinnForum configuration for SCS.15 """
+    # Create the actual config for client cert/key path
+    
+    config = {
+        'clientCert': self.getCertFilename("client_inapplicable.cert"),
+        'clientKey': self.getCertFilename("client.key")
+    }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_SCS_15_default_config)
+  def test_WINNF_FT_S_SCS_15(self,config_filename):
+    """Certificate with inapplicable fields presented during registration. 
+    
+    Checks that SAS UUT response with tls connection succedded and response should be 104.    
+    """
+    self.SasReset()
+    config = loadConfig(config_filename)
+    
+    # load device certs and device_a from config file
+    device_a_cert = config['client_cert_path']
```

Comment:
Same: keys don't match those defined in generate_SCS_15_default_config()

Reply:
Thanks   for catching this. The keys have been updated correctly.